package.json

```diff
@@ -1,22 +1,78 @@
-{
-	"name": "coffee-loader",
-	"version": "0.7.3",
-	"author": "Tobias Koppers @sokra",
-	"description": "coffee loader module for webpack",
-	"dependencies": {
-		"loader-utils": "^1.0.2"
-	},
-	"peerDependencies": {
-	    "coffeescript": ">= 1.8.x"
-	},
-	"repository": {
-		"type": "git",
-		"url": "git@github.com:webpack/coffee-loader.git"
-	},
-	"licenses": [
-		{
-			"type": "MIT",
-			"url": "http://www.opensource.org/licenses/mit-license.php"
-		}
-	]
-}
+{
+  "name": "coffee-loader",
+  "version": "0.7.3",
+  "author": "Tobias Koppers @sokra",
+  "description": "coffee loader for webpack",
+  "main": "dist/cjs.js",
+  "files": [
+    "dist"
+  ],
+  "engines": {
+    "node": ">= 4.3 < 5.0.0 || >= 5.10"
+  },
+  "scripts": {
+    "start": "yarn run build -- -w",
+    "build": "cross-env NODE_ENV=production babel src -d dist --ignore 'src/**/*.test.js'",
+    "clean": "del-cli dist dmd jsdoc-api jest test/builds",
+    "docs": "jsdoc2md src/index.js > LOADER.md",
+    "lint": "eslint --cache src test",
+    "lint-staged": "lint-staged",
+    "prebuild": "yarn run clean",
+    "prepublish": "yarn run build",
+    "release": "yarn run standard-version",
+    "security": "nsp check",
+    "pretest": "yarn run build && node test/webpack.build.js",
```

Comment:
Yep agreed, the loader works that was the baseline for the moment I will try to out a different approach later (it's 'planned' since a few days now:)) to start webpack in the test instead of making a 'test bundle'. Is there anything shipped already somewhere ?